step-definitions/Hotels/hotelsSD.js

```diff
@@ -0,0 +1,225 @@
+const { Given, When, Then } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const Feedbackpage = require("../../Pages/Hotels/Feedbackpage");
+const Homepage = require("../../Pages/Hotels/Homepage");
+const SignInpage = require("../../Pages/Hotels/SignInpage");
+const SignUppage = require("../../Pages/Hotels/SignUppage");
+const TermsConditionspage = require("../../Pages/Hotels/TermsConditionspage");
+
+const hpage = new Homepage();
+const spage = new SignInpage();
+const supage = new SignUppage();
+const tcpage = new TermsConditionspage();
+const fpage = new Feedbackpage();
+// T-18 -->
+Given(/^I am on hotels landing page$/, async function(){
+    await browser.url('/');
+    await browser.pause(2000);
+});
+When(/^I click on Travelers$/, async function(){
+    await hpage.clickOnTravelers();
+    await browser.pause(2000);
```

Comment:
Instead of hard coded wait on SD, you can use dynamic wait like waitFor() or waitUntil() inside respective function on POM. In this case you can call wait function at end of clickOnTravelers() or at beginning of setNumOfAdults().